src/Settings.ts

```diff
@@ -103,6 +105,7 @@ export default class EditInNeovimSettingsTab extends PluginSettingTab {
           .onChange(async (value) => {
             this.plugin.settings.supportedFileTypes = value.split(" ");
             await this.plugin.saveSettings();
+            await this.plugin.updateNeovimInstance();
```

Comment:
You mentioned in [this](https://github.com/TheseusGrey/edit-in-neovim/pull/13/files#r1915348737) comment about settings requiring a restart.

But you haven't added it to the main setting that would require it (the `listenOn` setting that specifies the socket or ip:port)